Alert rule filters and settings updates (#3513)
getsentry/sentry-docs: src/docs/product/alerts-notifications/alert-settings.mdx

## Rules

> The “Alert Rules” tab displays your alert rules, along with their current status, project, team, and creation date. The list is filtered so that only alerts associated with the teams you're a member of are displayed. You can change this using the "Filter" button. The name of the button changes to reflect the number of filters applied, so when you first navigate to the page, it's labeled "2 Active Filters".

Reviewer: Is the last sentence an example (if so, we should say "for example")?

Author: AFAIK that's the default number of filters for everyone (updated screenshot coming), but @iProgramStuff should confirm.

Reviewer 2: It is the same. It's still an example though, right?

Author: I'm trying to explain the default behaviour, mostly because I'm worried about folks coming to the page, expecting to see a button labeled "Filter" and not seeing that, but instead seeing a button that says "2 Active Filters." How does this work @iProgramStuff and @PeloWriter 

```suggestion
The “Alert Rules” tab displays your alert rules, along with their current status, project, team, and creation date. The list is filtered so that only alerts associated with the teams you're a member of, as well as ones that aren't associated with any team are displayed. You can change this using the "Filter" button. The name of the button changes to reflect the number of filters applied, so when the default filters are selected, it's labeled "2 Active Filters".
```

> The “Alert Rules” tab displays your alert rules, along with their current status, project, team, and creation date. The list is filtered so that only alerts associated with the teams you're a member of are displayed. You can change this using the "Filter" button. The name of the button changes to reflect the number of filters applied, so when you first navigate to the page, it's labeled "2 Active Filters".

Reviewer 2: ```suggestion
The “Alert Rules” tab displays your alert rules, along with their current status, project, team, and creation date. By default, the list is filtered so that only alerts associated with the teams you're a member of, as well as ones that aren't associated with any team, are displayed. You can change this using the "Filter" button. The name of the button changes to reflect the number of filters applied, so when you first navigate to the page, it's labeled "2 Active Filters".
```

Reviewer 2: Adding `by default` read a lot better to me? The information is accurate though :)

> The “Alert Rules” tab displays your alert rules, along with their current status, project, team, and creation date. By default, the list is filtered so that only alerts associated with the teams you're a member of, as well as ones that aren't associated with any team, are displayed. You can change this using the "Filter" button. The name of the button changes to reflect the number of filters applied, so when you first navigate to the page, it's labeled "2 Active Filters".

Reviewer 3: ```suggestion
The “Alert Rules” tab displays your alert rules, along with their current status, project, team, and creation date. By default, the list is filtered so that only alerts associated with the teams you're a member of, as well as ones that aren't associated with any team, are displayed. You can change this using the filter button.
```

> If you have [Performance Monitoring](/product/performance/) configured for your organization, the **Alerts** page also displays a “History” tab where you can find a list of triggered alerts, along with visual graphs, associated projects, and the last time the alert was triggered. In the “History” tab, the default view displays unresolved alerts.

Reviewer 3: ```suggestion
The **Alerts** page also displays a “History” tab where you can find a list of [Metric Alerts](/product/alerts-notifications/metric-alerts/), along with visual graphs, associated projects, and the last time the alert was triggered. In the “History” tab, the default view displays unresolved metric alerts.
```

## Settings

You can customize the settings for an alert, as well as your own personal alert settings.

### Alert Settings

With alert settings, you can configure alert email templates and set minimum and maximum delivery intervals for alerts. Access alert settings by navigating to **[Project] > Settings > Alerts**.

#### Digests

> The digests feature works only for **issue alert emails** and limits alerts across issues within a project. This project-level setting allows you to batch issue alerts to limit the total number of emails people in your organization receive for that project. Use the sliders to control the frequency.

Reviewer 3: ```suggestion
The digests feature works only for **issue alert emails** and limits alerts across issues within a project. This project-level setting allows you to batch issue alerts to limit the total number of emails sent for that project. Use the sliders to control the frequency.
```
